compose.yml

```diff
@@ -11,6 +11,7 @@ x-app: &app
   tmpfs:
     - /tmp
     - /app/tmp/pids
+    - /rails/tmp/pids
```

Comment:
I looked into that first but this director wasn't present on my local filesystem and I couldn't check into the server container because by the time I could `exec` into it, the container would have crashed and stopped.

So, I just added it in `tmpfs`. Let me know if I should dig more into this so that we don't have to add `/rails/tmp/pids` to `tmpfs`

Reply:
So `/` is mounted at `rails/` in the docker container. So if you saw an error like this:

```
web-1       | A server is already running (pid: 1, file: /rails/tmp/pids/server.pid).
```

That file would be in `rm -rf tmp/pids/server.pid`.